<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newProjectDialog</class>
 <widget class="QDialog" name="newProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Project</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0,0,0,0">
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Project Name</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Project Directory</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>247</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Create New Project</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="6">
    <widget class="QLineEdit" name="txt_name"/>
   </item>
   <item row="2" column="2" colspan="5">
    <widget class="QLineEdit" name="txt_dir">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>240</red>
           <green>240</green>
           <blue>240</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>240</red>
           <green>240</green>
           <blue>240</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>240</red>
           <green>240</green>
           <blue>240</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="4">
    <widget class="QPushButton" name="btn_cancel">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>150</height>
      </size>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="7" column="5">
    <widget class="QPushButton" name="btn_create">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Create Project</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="5">
    <widget class="QLineEdit" name="txt_sym_address">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>240</red>
           <green>240</green>
           <blue>240</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>240</red>
           <green>240</green>
           <blue>240</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>240</red>
           <green>240</green>
           <blue>240</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Symbolic Layout</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QToolButton" name="btn_openDirectory">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Open Folder</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QToolButton" name="btn_symbLayout">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Open File</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
